paris-paristechprofile/week-2/js-racer-game-lab/Paris' Code/app.js

```diff
@@ -0,0 +1,58 @@
+window.onload = function () {
+  player = {
+    blue: {
+      div: document.querySelector(".blueBox"),
+      name: "Blue"
+    },
+    red: {
+      div: document.querySelector(".redBox"),
+      name: "Yellow"
+    },
+  };
+};
+
+//activate button listening
+const go = document.querySelector(".ready").addEventListener("click", function() {
+//show a user a GO!
+    document.getElementsByTagName("h2")[0].innerText = "GO!";
+});
+
```

Comment:
I like the use of const in your document, but I can tell that this code is a majority of the solution code. I would like it when you have time to go back and work through the logic of the game differently. I can tell from some of your other code that you can produce more advanced logic. Would love to see your style shine. ⭐️